File: app/src/main/java/com/petproject/andy/bcstorage/data/database/Barcode.java
package com.petproject.andy.bcstorage.data.database;


import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "Barcode")
public class Barcode {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    public long id;



    @ColumnInfo(name = "Barcode")
    public String barcode;

    @ColumnInfo(name = "productName")
    public String productName;

    @ColumnInfo(name = "Location")
    public String location;

    @ColumnInfo(name = "Quantity")
    public int quantity;

}

File: app/src/main/java/com/petproject/andy/bcstorage/data/database/Barcode.kt
package com.petproject.andy.bcstorage.data.database


import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "Barcode")
class Barcode {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    var id: Long = 0


    @ColumnInfo(name = "Barcode")
    var barcode: String? = null

    @ColumnInfo(name = "productName")
    var productName: String? = null

    @ColumnInfo(name = "Location")
    var location: String? = null

    @ColumnInfo(name = "Quantity")
    var quantity: Int = 0

}
